steps/image-overlayer/1.0.5/step.yml

```diff
@@ -0,0 +1,86 @@
+title: Image overlayer
```

Comment:
Based on the [Step Title conventions](https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title) `Add image overlay` (or something similar) would be a better title.

Reply:
I took the point, thank you 🙂